### PR TITLE
NIFI-3291: Continue addressing issues from jQuery upgrade

### PR DESCRIPTION
**Pull request by mcgilman**
NIFI-3291:
- Fixing overflow calculation to determine if scrollbars are necessary.
- Fixing styles with jquery ui slider usage.

### CODE REVIEW

**Comment by scottyaslan** [comment]
Reviewing....

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/provenance/nf-provenance-table.js [hunk 1]
```diff
@@ -617,7 +617,7 @@
 
                 // adjust the field width for a potential scrollbar
                 var searchFieldContainer = $('#searchable-fields-container');
-                if (searchFieldContainer.get(0).scrollHeight > searchFieldContainer.innerHeight()) {
+                if (searchFieldContainer.get(0).scrollHeight > Math.round(searchFieldContainer.innerHeight())) {
```
Comment (scottyaslan) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/provenance/nf-provenance-table.js]: @mcgilman I think this whole block of code is dead. The widths of the inputs for the "Search Events" dialog of the "Data Provenance" shell's $('#searchable-fields-container') are defined by CSS (in provenance.css line #141) and the widths being applied with this code are causing those inputs not to fill their available space.

Comment (mcgilman) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/provenance/nf-provenance-table.js]: Yep, agreed. Will update.

**Comment by scottyaslan** [comment]
Thanks @mcgilman this has been merged to master.

**Pull request closed without merging**